docs/modules/ROOT/pages/partials/con_hermetic_verification.adoc

```diff
@@ -1,16 +1,17 @@
 .Verification
 
-. On your application, go to the *Activity > Pipeline runs* tab.
+. From the *Applications* view, go to *Activity > Pipeline runs*.
```

Comment:
Does this need to specify Applications view in <product>, so they know it isn't in GH?

Reply:
really good catch. i think so, yes. adding.